src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/features.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import QuestionTooltip from 'app/components/questionTooltip';
+import space from 'app/styles/space';
+import {
+  CandidateDownload,
+  CandidateDownloadStatus,
+  CandidateFeatures,
+} from 'app/types/debugImage';
+
+import NotAvailable from '../../notAvailable';
+
+import {getCandidateFeatureLabel} from './utils';
+
+type Props = {
+  download: CandidateDownload;
+};
+
+function Features({download}: Props) {
+  if (download.status !== CandidateDownloadStatus.OK) {
+    return <NotAvailable />;
+  }
+
+  const features = Object.entries(download.features).filter(([_key, value]) => value);
+
+  if (!features.length) {
+    return <NotAvailable />;
+  }
+
+  return (
+    <Wrapper>
+      {Object.entries(download.features)
+        .filter(([_key, value]) => value)
```

Comment:
Is this the same as `features` variable?

Reply:
I dont think so

Reply:
yes